DataBiosphere/leonardo
# IA-3640 update PD status properly when runtime creation fails during monitoring

https://broadworkbench.atlassian.net/browse/IA-3640

Similar to https://github.com/DataBiosphere/leonardo/pull/3048

Tested locally
1. Run leo locally
2. Create a VM with `us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.0.2`
3. The creation will timeout, and go to cloud env page on UI, the associated persistent should not show up on this page as `Creating`

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:


In all cases:
